frontend/src/pages/servers/index.tsx

```diff
@@ -38,27 +62,60 @@ const getStats = (servers: Array<Essential>): Stats => ({
   servers: servers.length,
 });
 
-const Row = ({ s }: { s: Essential }) => (
-  <li className="hover-bg-black-10 lh-copy pa2 ph0-l bb b--black-10">
-    <Link href={"/servers/" + s.ip}>
-      <a className="link black flex items-center justify-between">
-        <div className="overflow-hidden">
-          <span className="db black-70 measure truncate">{s.hn}</span>
-          <span className="db black-30 f6 measure truncate">{s.gm}</span>
-        </div>
+const Row = ({ s }: { s: Essential }) => {
+  const { onCopy, hasCopied } = useClipboard(s.ip);
 
-        <div className="pr2 tr dn di-ns flex-shrink-0">
-          <div className="black-70">
-            {s.pa ? <span>🔐</span> : null} <span>{s.ip}</span>
-          </div>
-          <div className="db black-30 f6 measure">
-            {s.pc}/{s.pm} playing now
-          </div>
-        </div>
-      </a>
-    </Link>
-  </li>
-);
+  return (
+    <Box
+      style={{
+        border: "1px solid #8477B7",
+        borderRadius: "8px",
+        padding: "1em",
+        marginBottom: "1em",
+      }}
+      boxShadow="base"
+    >
+      <Box>
+        <Link href={"/servers/" + s.ip}>
+          <Heading
+            fontSize={"xl"}
+            style={{ marginTop: "0" }}
+            _hover={{ textDecor: "underline", cursor: "pointer" }}
+          >
+            {s.hn}
+          </Heading>
+        </Link>
+
+        <Flex justifyContent="space-between" alignItems="center" my="0.4em">
+          <VStack align="left">
+            <Text style={{ marginTop: "0" }}>{s.gm}</Text>
+            {/* <Text style={{ marginTop: "0" }}>{"website"}</Text> */}
+          </VStack>
+          <Flex flexDirection="row" alignItems="center" gridGap=".5em">
+            <Text fontWeight={"bold"} fontSize="2xl" style={{ marginTop: "0" }}>
+              {s.pc}/{s.pm}
+            </Text>
+            <Text style={{ marginTop: "0" }}>players</Text>
+          </Flex>
+        </Flex>
+        <HStack>
```

Comment:
This little IP+Copy could be its own component too, I could see it being useful elsewhere